### PR TITLE
Remove charts theme code

### LINKED ISSUE #531: Remove charts theme code
OUI doesn't provide any sort of charting. As a carryover from EUI, we still have a [`src/themes/charts`](https://github.com/opensearch-project/oui/tree/a657d1f16be5953b89b38fc085221f0b39fc18f8/src/themes/charts) folder. That can be deleted assuming that it's not being used anywhere.

### PR DESCRIPTION
**Pull request by tarunsamanta2k20**
### Description
Removed the charts folder with two files (theme.scss and themes.ts).
 
### Issues Resolved
#531 
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] All tests pass
  - [x] `yarn lint`
  - [x] `yarn test-unit`
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by KrooshalUX** [comment]
@tarunsamanta2k20 Can you please describe what your PR resolves?

**Comment by BSFishy** [comment]
Thanks for the contribution! There are a few things that I would like to be completed before we review this PR. Specifically, could the description be filled out, specifically the `Description` and `Issues Resolved` sections?

Also, it looks like DCO didn't pass. Do you think you could sign your commit? Related links:
 - https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin
 - https://www.atlassian.com/git/tutorials/rewriting-history

**Comment by SergeyMyssak** [comment]
Hello @tarunsamanta2k20, in order to merge this PR, you need to do the following steps:
1) In the `Description` section, write what was done in this PR, for example: 
> Removed the `charts` folder with two files (`theme.scss` and `themes.ts`).
2) In the `Issues Resolved` section, simply paste the link to the problem you solved, for example:
> https://github.com/opensearch-project/oui/issues/531
3) Run `yarn lint` and `yarn test-unit` commands locally with your changes and then select the appropriate checkboxes. To select a checkbox you need to write `x` inside `[]`, like this `[x]`.
4) You need to change your commit message and re-commit your changes with the `--signoff` flag, please make sure your GitHub email is public (you can check it in your GitHub settings).

You can copy these commands and run them:
```
git reset --soft HEAD~1
git commit -m "Remove theme folder for charts" --signoff
```
After that, you can select the `Commits are signed per the DCO using --signoff` checkbox.

---
FYI: @tarunsamanta2k20, please don't forget to remove `charts` folder, I see that there is an empty folder left in the code.

Thanks for contributing!

**Comment by SergeyMyssak** [comment]
Hello @tarunsamanta2k20, I see that you changed the description of the issue! Well done! Now just need to pass the DCO pipeline. Please try to run these commands:
```
git reset --soft HEAD~1
git commit -m "Remove theme folder for charts" --signoff
```
And don't forget to push this updated commit to your branch:
```
git push origin tarun_samanta/#531 --force
```
Just a little bit left! Thanks for your changes!

**Comment by tarunsamanta2k20** [comment]
> Hello @tarunsamanta2k20, I see that you changed the description of the issue! Well done! Now just need to pass the DCO pipeline. Please try to run these commands:
> 
> ```
> git reset --soft HEAD~1
> git commit -m "Remove theme folder for charts" --signoff
> ```
> 
> And don't forget to push this updated commit to your branch:
> 
> ```
> git push origin tarun_samanta/#531 --force
> ```
> 
> Just a little bit left! Thanks for your changes!

@BSFishy done!

**Comment by joshuarrrr** [comment]
@tarunsamanta2k20 The linter check is failing because there's still a reference to one of the deleted files that needs to be removed: https://github.com/opensearch-project/oui/blob/main/src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js#L22

and
https://github.com/opensearch-project/oui/blob/main/src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js#L251

**Comment by tarunsamanta2k20** [comment]
@joshuarrrr let me check once.

**Comment by tarunsamanta2k20** [comment]
> @tarunsamanta2k20 The linter check is failing because there's still a reference to one of the deleted files that needs to be removed: https://github.com/opensearch-project/oui/blob/main/src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js#L22
> 
> and https://github.com/opensearch-project/oui/blob/main/src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js#L251

@joshuarrrr let me check once.

**Comment by tarunsamanta2k20** [comment]
@joshuarrrr shall I remembered ve entire block of code under those imports?

**Comment by tarunsamanta2k20** [comment]
@joshuarrrr  Let me check once.

File: src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js [hunk 1]
```diff
@@ -19,7 +19,6 @@ import {
   DataGenerator,
 } from '@elastic/charts';
 
-import { OUI_CHARTS_THEME_LIGHT } from '../../../../src/themes/charts/themes';
```
Comment (BSFishy) [inline src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js:22]:
```suggestion
```

The linter is throwing an error at this

Comment (tarunsamanta2k20) [reply src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js:22]: @BSFishy  I need to add a space.

Comment (BSFishy) [reply src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js:22]: It looks like the issue is that there are 2 empty lines there, which gets caught by our linter. Removing one of them should fix the issue

Comment (tarunsamanta2k20) [reply src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js]: @BSFishy Done!.

**Review by BSFishy — APPROVED** [review]
LGTM

**Review by joshuarrrr — APPROVED** [review]

**Comment by joshuarrrr** [comment]
@BSFishy Does this need to be backported?

**Comment by BSFishy** [comment]
> @BSFishy Does this need to be backported?

This is breaking. No backport needed. (Also of note, I'm pretty sure this is incompatible with Dashboards)

**Comment by tarunsamanta2k20** [comment]
@BSFishy please follow me on GITHUB.

**Pull request merged**